PxCs/Extensions/SoundExtension.cs

```diff
@@ -0,0 +1,302 @@
+using System;
+using System.Text;
+using System.IO;
+using System.Collections.Generic;
+
+namespace PxCs.Extensions
+{
+    internal enum WAVE_FORMAT
+    {
+        UNKNOWN,
+        PCM,
+        ADPCM,
+        IMA_ADPCM = 0x11,
+    }
+
+    internal class Ima
```

Comment:
Can you rename Ima or at least tell in a comment what it means? I don't know this acronym.

Reply:
Renamed to IMAADPCMDecoder